### ISSUE TITLE
Discussion: Format to visually explain concepts

### ISSUE THREAD

**Issue by cassiozen** [cassiozen/ReactCasts#23]
Hey Guys, I'm experimenting with different ways of explaining concepts visually.

Because what usually happens while I'm explaining something is that the video is showing a blank editor. This happened in almost all episodes so far.

This is my first try: If you have the time to take a look, I would love to hear your opinion:

(Disclaimer: The actual content I was explaining is not important - I just improvised something to show the new visual)

https://youtu.be/soMI7BAVtLU

(I'm specifically inviting all of you who have contributed so far: @markerikson, @neeldeep, @jpollard-cs, @asifmom, @pawel2105, @czanella, @zsherman, @thelastnode and @aconanlai)

**Comment by cassiozen** [comment]
By the way, I'm using https://www.text2mindmap.com for this (with a custom stylesheet).

**Comment by zsherman** [comment]
this is awesome!

**Comment by neeldeep** [comment]
Looks good. 👍 
Highly ambitious PR...using this to explain some of workings of Fiber

**Comment by aconanlai** [comment]
personally, i'm not sure if it explains things any better for me (i find your videos very easy to follow as is), although others might be more visual learners

**Comment by pawel2105** [comment]
Hey @cassiozen 

It could work. Definitely try giving it a shot in a couple of your up-coming videos. Maybe as a right-floated aside? I would probably advise that you remove the bounciness because overt distractions will prevent mental intake of the concepts you're teaching.

Yeah this might be cool. My philosophy is that any additional approach or perspective you can provide when teaching just provides a higher probability that concepts will "click" in someone's head. And as long as any additional perspectives don't overburden someone's mind, they're always net-positive _just in case_ they help someone incrementally.

Anyway, good luck and I love the innovative thinking!

Pawel

**Comment by thelastnode** [comment]
For some of the more intricate concepts, having visuals on top of code sounds like a great idea! This might be a situation where slides work better than something like text2mindmap, since you can set up the layout ahead of time, and include any type of content you like.

**Comment by cassiozen** [comment]
Thanks everybody for helping out! I think overall this is a "Go", right?

@zsherman, @neeldeep, @aconanlai: Thanks for your input!

@pawel2105: I tried to make it less distractive and "bouncy", but there so much I can do by injecting custom scripts on the service. (I Don't want to build a new one NOR reverse engineer the project).

@thelastnode: To be honest I don't like to use slides on the screencast. I tried, but it looks, I don't know, too static...
